### DISCUSSION TITLE
My homepage route is SSR / client-only mode

### DISCUSSION THREAD

**Discussion by jackcunningham** [nuxt/nuxt#19720] [Questions (Nuxt 2)]
All the pages including dynamic routes in my site are statically generated (SSG), when you view source you correctly see all the markup as expected. But for some reason the homepage route is in SSR/client-only mode, ie when you view source there is no content in the markup, just the scripts and if you disable JS you get a blank page.

The app is in Netlify and I can see an error during the deploy
`Error generating route "/ ": This page could not be found`.
I'm not sure if the space after the slash is significant?

My nuxt-config has no special settings other than interval/concurrency

```
  target: "static",
  ...
  generate: {
    interval: 500,
    concurrency: 30,
    routes: [
      '/reports/secret-pages/',
      '/page/form-thanks/'
    ]
  }

```

These versions are installed in `nuxt.config.js`
```
"dependencies": {
    "@nuxtjs/apollo": "^4.0.1-rc.5",
    "nuxt": "^2.15.7",
```

I've tried moving the homepage to a different route eg `/test/index.vue/` and SSG kicks in properly again, so there is no problem with the code in the page. Likewise I can change my homepage to a basic template like below and it still renders in client-only mode - it treats whatever I put in /index.vue as client-only.
```
<template>
    <div>
        <h1>Test</h1>
        <p>This is still in client only mode</p>
    </div>
</template>
```

How can i make sure my homepage statically generates?

**Comment by AndreasMueck** [comment]
This problem was solved by the thread author himself.
[https://stackoverflow.com/questions/75665234/nuxt-js-homepage-is-reverting-to-ssr-client-only-mode](https://stackoverflow.com/questions/75665234/nuxt-js-homepage-is-reverting-to-ssr-client-only-mode)
